apps/site/assets/ts/hooks/useSchedules.ts

```diff
@@ -36,14 +38,21 @@ const fetchData = async (url: string): Promise<ScheduleData[]> =>
 const useSchedulesByStop = (
   stopId: string
 ): FetchState<ScheduleWithTimestamp[]> => {
+  const [updateDate, setUpdateDate] = useState(Date.now());
+  useInterval(() => {
+    setUpdateDate(Date.now());
+  }, 15000);
+
   const { data, error } = useSWR<ScheduleData[]>(
     `/api/stops/${stopId}/schedules?last_stop_departures=false&future_departures=true`,
     fetchData
   );
   if (error) {
     return { status: FetchStatus.Error };
   }
-  const parsedData = data?.map(d => parse(d));
+  const parsedData = data
```

Comment:
question: The number of schedules is no longer an issue because there's no longer a react hook for each right?

Reply:
@i0sea yes...
(also I had no special reason for choosing 15 seconds, just felt nice and frequent and avoids waiting 59 seconds for a "<1 minute" to go away 😅 )